### PR TITLE
Clarify statement on backup-and-restore-using-dumpling page

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

1. The statement to install lightning wasn't correct
2. There is no need to "start" dumpling or lightning
3. Make it clear that the "50 MB" is just a recommendation about when to use Dumpling/Lightning and when to use BR.
4. Format SQL statements so no horizontal scrolling is needed. Also use `FORMAT_BYTES()` instead of manual conversion to MiB. And remove the `WHERE TABLE_SCHEMA=...` to make the statements copy-pasteable.   

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: backup-and-restore-using-dumpling-lightning.md [hunk 1]
```diff
@@ -7,20 +7,20 @@ summary: Learn how to use Dumpling and TiDB Lightning to back up and restore ful
 
 This document introduces how to use Dumpling and TiDB Lightning to back up and restore full data of TiDB.
 
-If you need to back up a small amount of data (for example, less than 50 GB) and do not require high backup speed, you can use [Dumpling](/dumpling-overview.md) to export data from the TiDB database and then use [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) to import the data into another TiDB database. For more information about backup and restore, see [TiDB Backup & Restore Overview](/br/backup-and-restore-overview.md).
+If you need to back up a small amount of data (for example, less than 50 GiB) and do not require high backup speed, you can use [Dumpling](/dumpling-overview.md) to export data from the TiDB database and then use [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) to restore the data into another TiDB database. To backup larger databases the recommended method is to use BR, for more information about backup and restore with BR, see [TiDB Backup & Restore Overview](/br/backup-and-restore-overview.md). Note that Dumpling can be used to export large databases, but BR is better suited tool for that.
```
Comment (hfxsd) [inline backup-and-restore-using-dumpling-lightning.md:10]:
```suggestion
If you need to back up a small amount of data (for example, less than 50 GiB) and do not require high backup speed, you can use [Dumpling](/dumpling-overview.md) to export data from the TiDB database and then use [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) to restore the data into another TiDB database. 

To back up larger databases, the recommended method is to use [BR](/br/backup-and-restore-overview.md). Note that Dumpling can be used to export large databases, but BR is a better tool for that.
```

**Review by okJiang — APPROVED** [review]

**Review by Frank945946 — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

File: backup-and-restore-using-dumpling-lightning.md [hunk 2]
```diff
@@ -7,20 +7,22 @@ summary: Learn how to use Dumpling and TiDB Lightning to back up and restore ful
...
-    tiup install dumpling && tiup dumpling
+    tiup install dumpling
     ```
 
-- Install and start TiDB Lightning:
+- Install TiDB Lightning:
 
     ```shell
-    tiup install tidb lightning && tiup tidb lightning
+    tiup install tidb tidb-lightning
```
Comment (Oreoxmt) [inline backup-and-restore-using-dumpling-lightning.md:25]:
```suggestion
    tiup install tidb-lightning
```

File: backup-and-restore-using-dumpling-lightning.md [hunk 3]
```diff
@@ -41,14 +43,35 @@ If you need to save data of one backup task to the local disk, note the followin
 - Dumpling requires a disk space that can store the whole data source (or to store all upstream tables to be exported). To calculate the required space, see [Downstream storage space requirements](/tidb-lightning/tidb-lightning-requirements.md#storage-space-of-the-target-database).
 - During the import, TiDB Lightning needs temporary space to store the sorted key-value pairs. The disk space should be enough to hold the largest single table from the data source.
 
-**Note**: It is difficult to calculate the exact data volume exported by Dumpling from MySQL, but you can estimate the data volume by using the following SQL statement to summarize the `data-length` field in the `information_schema.tables` table:
+**Note**: It is difficult to calculate the exact data volume exported by Dumpling from MySQL, but you can estimate the data volume by using the following SQL statement to summarize the `DATA_LENGTH` field in the `information_schema.tables` table:
```
Comment (Oreoxmt) [inline backup-and-restore-using-dumpling-lightning.md:46]: Please update the "`data-length` field" in tidb-lightning/tidb-lightning-requirements.md and migrate-large-mysql-to-tidb.md

Comment (hfxsd) [reply backup-and-restore-using-dumpling-lightning.md:46]: Done

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**